models/clan.go

```diff
@@ -53,6 +55,52 @@ type Clan struct {
 	DeletedAt        int64                  `db:"deleted_at" json:"deletedAt" bson:"deletedAt"`
 }
 
+// MostRecentFirst is the constant "mostRecentFirst"
+const MostRecentFirst string = "mostRecentFirst"
```

Comment:
why not asc and desc?

Reply:
This way the value of the input is more semantic for the user. The code later translates to a valid SQL query order.

Reply:
Changed to "newest" and "oldest".